packages/core/src/ceramic.ts

```diff
@@ -361,23 +379,19 @@ class Ceramic implements CeramicApi {
   /**
    * @returns An array of the CAIP-2 chain IDs of the blockchains that are supported for anchoring
    * documents.
-   *
-   * Caches the result after the first time it is requested
    */
   async getSupportedChains(): Promise<Array<string>> {
-    if (this._supportedChains) {
-      return this._supportedChains
-    }
+    const chainIds = Object.keys(this.context.anchorServices)
 
-    // Fetch the chainId from the anchor service and cache the result
-    if (!this.context.anchorService) {
-      throw new Error("No anchor service configured")
+    const index = chainIds.indexOf(DEFAULT_ANCHOR_SERVICE_CHAIN_ID);
+    if (index > -1) {
+      // set default chainId to be the first one
```

Comment:
Instead of doing this every time getSupportedChains() is called, maybe we could do it once in the constructor and store the properly ordered list of supported chainIds?

Reply:
yep, we can do that 👍